Core/DISET/ServiceReactor.py

```diff
@@ -34,123 +34,126 @@
 from DIRAC.ConfigurationSystem.Client.Helpers import Registry
 from DIRAC.ConfigurationSystem.Client import PathFinder
 
-class ServiceReactor( object ):
 
-  __transportExtraKeywords = { 'SSLSessionTimeout' : False, 
-                               'IgnoreCRLs': False, 
-                               'PacketTimeout': 'timeout' }
+class ServiceReactor(object):
 
-  def __init__( self ):
+  __transportExtraKeywords = {'SSLSessionTimeout': False,
+                              'IgnoreCRLs': False,
+                              'PacketTimeout': 'timeout',
+                              'SocketBacklog': False,
```

Comment:
Shouldn't this be a `int` value?

Reply:
No... This dict is just a lookup dictionary such that the "PacketTimeout" option is renamed "timeout" .....